[2022-06-15 22:29:03,430] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: say_hi.get_name_task 2022-06-15T22:29:00.655279+00:00 [queued]>
[2022-06-15 22:29:03,457] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: say_hi.get_name_task 2022-06-15T22:29:00.655279+00:00 [queued]>
[2022-06-15 22:29:03,458] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-06-15 22:29:03,458] {taskinstance.py:1068} INFO - Starting attempt 1 of 2
[2022-06-15 22:29:03,470] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-06-15 22:29:03,540] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): get_name_task> on 2022-06-15T22:29:00.655279+00:00
[2022-06-15 22:29:03,565] {standard_task_runner.py:52} INFO - Started process 298 to run task
[2022-06-15 22:29:03,579] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'say_hi', 'get_name_task', '2022-06-15T22:29:00.655279+00:00', '--job-id', '7', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/username.py', '--cfg-path', '/tmp/tmprye7261f', '--error-file', '/tmp/tmp9r7sbef0']
[2022-06-15 22:29:03,580] {standard_task_runner.py:77} INFO - Job 7: Subtask get_name_task
[2022-06-15 22:29:03,719] {logging_mixin.py:104} INFO - Running <TaskInstance: say_hi.get_name_task 2022-06-15T22:29:00.655279+00:00 [running]> on host c4c189ca1669
[2022-06-15 22:29:03,981] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=say_hi
AIRFLOW_CTX_TASK_ID=get_name_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-15T22:29:00.655279+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-15T22:29:00.655279+00:00
[2022-06-15 22:29:03,982] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2022-06-15 22:29:03,983] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo Alma > /opt/***/dags/username.txt']
[2022-06-15 22:29:04,009] {subprocess.py:75} INFO - Output:
[2022-06-15 22:29:04,013] {subprocess.py:83} INFO - Command exited with return code 0
[2022-06-15 22:29:04,165] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=say_hi, task_id=get_name_task, execution_date=20220615T222900, start_date=20220615T222903, end_date=20220615T222904
[2022-06-15 22:29:04,316] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-15 22:29:04,416] {local_task_job.py:151} INFO - Task exited with return code 0
